DTS-STN/Alpha-Site
# Analytics script

# Description

I moved the analytics page bottom script on each page instead of the _document page.
I also changed some page titles to be consistent across the whole app.

## Acceptance Criteria

Check to see if the script is on each page and each page title has Service Canada Labs

## Test Instructions

1. Check to see if the script is on each page
2. Check to see if the titles have Service Canada Labs and the meta tag with "dc-terms.title" is the same

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
